runner/runner.py

```diff
@@ -92,6 +93,7 @@ def runner_arguments():
     parser.add_argument("-VMImageURL", default=None, help="The custom image resource URL, if you want the temlates to run on a custom image")
     parser.add_argument("-KeyVaultUrl", default=None, help="Azure Key vault to fetch secrets from, service principal must have access")
     parser.add_argument("-CleanUpResources", action="store_false")
+    parser.add_argument("-templateResourcesBranchFileName", default=None, help="Select the branch you want to pull your resources files from, default=master, current=Gets the branch you working on")
```

Comment:
isn't this just a branch name?

templateResourcesBranchFileName

but its called "branch file name"